src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+public class ColorSupplier {
+    private static String[] colors = new String[]{"yellow", "red", "blue", "green", "white"};
+
+    public static String getRandomColor() {
+        return colors[(int)(Math.random() * 5)];
```

Comment:
better to use `Random` class

Reply:
Using new Random.nextInt(AnyNumber) will lead to 0 in result, should I always add 1 in the end then?

Reply:
is it?
![image](https://user-images.githubusercontent.com/58442482/130076528-ca88b47c-ebcc-4fc6-ab0c-027bbb5335c0.png)